src/lxc/lxccontainer.h

```diff
@@ -867,6 +873,17 @@ struct lxc_container {
 	 */
 	int (*seccomp_notify_fd_active)(struct lxc_container *c);
 
+	/*!
+	 * \brief Set response receive timeout for LXC commands
+	 *
+	 * \param c Container
+	 * \param timeout Seconds to wait before returning false.
+	 *  (0 to wait forever).
```

Comment:
Should this be `-1` to wait forever and `0` to return immediately to keep in line with other socket APIs?

Reply:
I thought about that too. But I can't imagine a use case for that here. Technically, when we are reading from LXC command socket `recvmsg` always getting blocked on waiting command result from the LXC container control daemon. I.e. if user set `0` to return immediately then all LXC commands that use this timeout setting will always return immediately without waiting for result. I understand why this is useful for `wait`/`reboot`/`shutdown` cases, but not wider.

Reply:
Then block `0` but I still think that `-1` should indicate no timeout as is customary.

Reply:
> Then block `0` but I still think that `-1` should indicate no timeout as is customary.

sure!